frontend/src/components/UI/Header/Menu.jsx

```diff
@@ -26,6 +26,9 @@ export const Menu = () => {
                 <li className="usa-nav__primary-item">
                     <Link to="/agreements?filter=all-agreements">Agreements</Link>
                 </li>
+                <li className="usa-nav__primary-item">
+                    <Link to="/budget-lines?filter=all-budget-line-items">Budget Lines</Link>
```

Comment:
maybe keep this `/budget-lines` and let `react-router` redirect to the filters?

Reply:
oh and this will light up the active tab...

Reply:
Did we want start by pointing them to a narrower view first? ie: Main link would take them to "My Budgetlines" ?

Reply:
Not sure where we landed on this but there was some discussion on the topic. Right now it is set to all BLIs.